### PR TITLE
Revise README, remove "experimental PoC" descriptor

### PR DESCRIPTION
**Pull request by Cynerva**
Update the README to better reflect the current state of the Kube-OVN charm as production ready.

### CODE REVIEW

**Comment by Cynerva** [comment]
@evilnick Can you review this?

**Comment by Cynerva** [comment]
The integration test failure is caused by https://bugs.launchpad.net/bugs/1986856, which should be fixed in edge soon.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
-# Kube-OVN charm
+# Kube-OVN Charm
 
-This is an experimental proof-of-concept charm for using Kube-OVN with
-Charmed Kubernetes. This is a subordinate machine charm that relates to
-kubernetes-control-plane and kubernetes-worker via the kubernetes-cni
+Kube-OVN is a CNI implementation based on OVN that provides a rich
```
Comment (evilnick) [inline README.md:3]: should we maybe include a link to the kube-ovn homepage or overview as well? Maybe https://kubeovn.github.io/docs/v1.10.x/en/

**Review by evilnick — APPROVED** [review]
one suggestion in comments

File: README.md (hunk 1 above)
Comment (Cynerva) [reply README.md]: Good call, thanks

**Pull request merged**